exercises/bowling/canonical-data.json

```diff
@@ -224,5 +224,21 @@
       "previousRolls": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 7, 3]
     },
     "expected": {"error": "Score cannot be taken until the end of the game"}
+  }, {
+    "description": "cannot roll after bonus roll for spare",
+    "property": "roll",
+    "input": {
+      "previousRolls": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 7, 3, 2],
+      "roll": 2
+    },
+    "expected": {"error": "Cannot roll after game is over"}
+  }, {
+    "description": "cannot roll after bonus rolls for strike",
+    "property": "roll",
+    "input": {
+      "previousRolls": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 10, 3, 2],
```

Comment:
A strike in the 10th rewards the roller with two extra rolls.  I count 15 in the 10th.

Reply:
I have called these bonus rolls, the test description calls them fill balls.

Reply:
Again, you counted correctly but neglected the `roll` input field.

Reply:
> I have called these bonus rolls, the test description calls them fill balls.

Other tests already referred to them as bonus rolls, so I remained consistent the the other tests. Perhaps another PR would be in order to make the description and tests consistent, but that is another issue.